packages/editor/src/context/useResizableEditableTable.ts

```diff
@@ -0,0 +1,84 @@
+import type { ColumnDef, RowSelectionState, SortingState } from '@tanstack/react-table';
+import { getCoreRowModel, getSortedRowModel, useReactTable } from '@tanstack/react-table';
+
+import { useState, useEffect } from 'react';
+import { deepEqual } from '../utils/equals';
+interface UseResizableEditableTableProps<TData> {
+  data: TData[];
+  columns: ColumnDef<TData>[];
+  onChange: (change: TData[]) => void;
+  EMPTY_DATA_OBJECT: TData;
```

Comment:
I know we have the convention that constants are written full uppercase. But here it is a parameter, so this one should definitely be lowercase.